js/app.js

```diff
@@ -193,15 +236,20 @@
    *   invoke this function expression passing 2 arguments
    *     event, 'light-green'
    */
-
+btn13.addEventListener('click', function(event){
+  setMyLightClass.bind(this)(event, 'light-green')
+})
 
   /*
-   * Add a click event listener to btn14
+   * Add a click event listener to btn14 
    * the handler method will be a function expression
    * that is the result of invoking the Function prototype method, bind
    * on the setMyLightGreen function
    */
 
+btn14.addEventListener('click', function(){
```

Comment:
So an anonymous function is not needed here. The result of the bind should be the event handler.